go-controller/pkg/networkmanager/nad_controller.go

```diff
@@ -153,69 +155,69 @@ func (nadController *nadController) syncAll() (err error) {
 	for _, nad := range existingNADs {
 		key, err := cache.MetaNamespaceKeyFunc(nad)
 		if err != nil {
-			klog.Errorf("%s: failed to sync %v: %v", nadController.name, nad, err)
+			klog.Errorf("%s: failed to sync %v: %v", c.name, nad, err)
 			continue
 		}
-		err = nadController.syncNAD(key, nad)
+		err = c.syncNAD(key, nad)
 		if err != nil {
-			return fmt.Errorf("%s: failed to sync %s: %v", nadController.name, key, err)
+			return fmt.Errorf("%s: failed to sync %s: %v", c.name, key, err)
 		}
 	}
 
 	return nil
 }
 
-func (nadController *nadController) sync(key string) error {
+func (c *nadController) sync(key string) error {
 	startTime := time.Now()
-	klog.V(5).Infof("%s: sync NAD %s", nadController.name, key)
+	klog.V(5).Infof("%s: sync NAD %s", c.name, key)
 	defer func() {
-		klog.V(4).Infof("%s: finished syncing NAD %s, took %v", nadController.name, key, time.Since(startTime))
+		klog.V(4).Infof("%s: finished syncing NAD %s, took %v", c.name, key, time.Since(startTime))
 	}()
 
 	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		klog.Errorf("%s: failed splitting key %s: %v", nadController.name, key, err)
+		klog.Errorf("%s: failed splitting key %s: %v", c.name, key, err)
 		return nil
 	}
 
-	nad, err := nadController.nadLister.NetworkAttachmentDefinitions(namespace).Get(name)
+	nad, err := c.nadLister.NetworkAttachmentDefinitions(namespace).Get(name)
 	if err != nil && !apierrors.IsNotFound(err) {
 		return err
 	}
 
-	return nadController.syncNAD(key, nad)
+	return c.syncNAD(key, nad)
 }
 
-func (nadController *nadController) syncNAD(key string, nad *nettypes.NetworkAttachmentDefinition) error {
+func (c *nadController) syncNAD(key string, nad *nettypes.NetworkAttachmentDefinition) error {
```

Comment:
I don't like making everything "c" or "controller" i think we have like 5 layers of "controller" abstractions that propagate down with pointers in different instances and it gets hard to tell what controller we are referencing. Maybe it's just me. In the future I'd prefer renaming things to be in its own PR separate from adding BGP functionality.

Reply:
Yeah, I prefer that too but it's not that easy in the environment we are working on. 
`c` is just the local receiver,  it's not being passed around.